botfront/imports/api/importExport/export.methods.js

```diff
@@ -12,30 +12,34 @@ import { generateErrorText } from './importExport.utils';
 
 
 if (Meteor.isServer) {
+    import { appLogger, addLoggingInterceptors } from '../../../server/logger';
+
+    const trainingAppLogger = appLogger.child({ fileName: 'export.methods.js' });
+    
     Meteor.methods({
         'exportProject'(apiHost, projectId, options) {
             check(apiHost, String);
             check(projectId, String);
             check(options, Object);
+            const appMethodLogger = trainingAppLogger.child({ userId: Meteor.userId(), methodName: 'exportProject', callingArgs: { apiHost, projectId, options } });
```

Comment:
Thanks, There is now a `getAppLoggerForMethod` and `getAuditLoggerForMethod`

Reply:
Great! But the third argument to `getAppLoggerForMethod` always takes 'Meteor.userId()' as the third argument. I know this will create strong coupling between your logger module and Meteor, but unless you can think of another possible value for the third argument, I would simply use 'Meteor.userId()' inside the 'getAppLoggerForMethod' and remove the third argument.

Reply:
Thanks! for now it only get the user Id through Meteor.userId(), however the mechanism to get the userId will be different when we will log graphQL queries. So I think we can keep userId as an argument

Reply:
I knew there was a good reason! ;-)